data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1875 @@ mission "Stone of our Fathers 5"
+			`	"Sorry to be late, you vould not believe all paper-vork they had for me. I think they aren't happy about Ice-Crawler."`
+			choice
+				`	"That's okay. I didn't mind waiting."`
+					goto waiting
+				`	"I was abducted by the Vice Planetary-Governor."`
+					goto abducted
+			label abducted
+			`	"That is no surprise." Hjlod says nonchalantly. "They alvays do thing like this. It matters not to us."`
+			`	You ask her if she's worried you've been compromised. That you might be working against the Skadenga's best interests.`
+			`	Hjlod shakes her head. "No. Future is set. Vhat vill, vill be. You are given task to tell us vhere to go, ve go vhere you say. The rest is just noise."`
+			label waiting
+			`	"Now", Hjlod says, "Tell me about Setstokkr, did you find it?"`
+			choice
+				`	"I found nothing definitive."`
+					goto nothing
+				`	"I think I found traces of it on Windblain."`
+					goto nothing
+				`	"I found a strange box similar to the Setstokkr on Asgard."`
+					goto box
+			label box
+			`	"Sounds like herbalist box. Ondurdis had one long before I vas born. Gave it to liar. It not important though, not Setstokkr."`
+			label nothing
+			`	"Sending big piece of vood through space probably not best idea Ondurdis had." Hjlod smirks.`
+			`	"Still, it does not matter so much what you find. Only how you felt. You saw the vorlds. Vhere should ve go?"`
+			choice
+				`	"Windblain."`
+					goto windblain
+				`	"Asgard."`
+					goto Asgard
+			label windblain
+			apply
+				set "Nifel"
+			`	"There is a nearly abandoned village called Hrithfjall next to a coastal God-Stone. Something about that place felt right. I believe that is where the Skadenga should migrate."`
+			`	Hjlod nods. "Hrithfjall sounds like good place. I vill tell Ondurdis. Vhen you were gone thing, or assembly, vas held. Ondurdis called in all Skadenga from all villages across Nifel. You must find transportation for 2,200 folk. That is what Onderduis told me to tell you."`
+			`	You frown and ask. "Did she provide money?"`
+			`	Hjlod shakes her head. "No. Ve could not afford it. You must go to Valhalla, and convince head of Transport Union to support us."`
+			`	Hjlod squeezes your shoulder. "Thank you for help. For finding us new home. Ve need it. I must report now to Ondurdis. Skade be vith you."`
+			`	She turns and leaves.`
+				accept
+			label Asgard
+			apply
+				set "Asgard"
+			`	"On Asgard there is a great market built around their God-Stone, and many new homes in need of people to fill them. The people there have lost their way and they need the Skadenga to remind them of who they were."`
+			`	Hjlod nods. "The Skadenga have alvays been guides, pathfinders. I vill tell Ondurdis. Vhen you were gone thing, or assembly, vas held. Ondurdis called in all Skadenga from all villages across Nifel. You must find transportation for 2,200 folk. That is what Onderduis told me to tell you."`
```

Comment:
Given the way this is presented in a somewhat positive light, and suddenly gives you an opinion that maybe they could have a calling here - the possibility of this outcome should really be foreshadowed earlier. Otherwise this option is entirely "doing it for the money" and suddenly "oh but here's a reason I've invented with no prior consideration to make you do it".

Reply:
I can change this. I just wanted to point out that this is said after you've made the decision. So the words here have no bearing on the decision making. The player should have already deduced for themselves what is the better option. This might give them hope that they can have their cake and eat it too, but they already know they've sold these people out.

Reply:
`The player should have already deduced for themselves what is the better option.`
Oh absolutely, but it puts words into the players mouth, without any control of the player. This is generally frowned upon.
For narrative reasons I think it is still important, but the player should have some warning of exactly how they're intending to lie to these people.
Perhaps maybe when you first fine the box and the godstone the player could muse to themselves something to the effect of, "Maybe the Skadenga could find a purpose here?" or something. Just _something_ to foreshadow the nature of the lie before you actually get to it.

Reply:
I have altered the text on the plaque to make it less condescending.

Reply:
Nice, I'm satisfied.